### PR TITLE
feat: added helm chart

### PR DESCRIPTION
**Pull request by wid**
Hello,

Please accept my contribution of a helm chart to deploy ralph on a kubernetes cluster.

Once you install the prerequisites (see README.md), it will start ralph along with nginx, mysql and redis with the helm install.

Thank you for this tool !

### CODE REVIEW

File: charts/ralph/values.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  port: 80
+
+ingress:
+  enabled: true
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: ralph-demo.allegro.tech
```
Comment (vi4m) [inline charts/ralph/values.yaml:50]: Not sure about that. Why this particular domain?

**Review by vi4m — CHANGES_REQUESTED** [review]

File: charts/ralph/values.yaml (hunk 1 above)
Comment (wid) [reply charts/ralph/values.yaml:50]: Well it's if you want to use it yourself.
You can put anything else there.

**Comment by vi4m** [comment]
I'm thinking about moving this configuration to the "contrib" directory (which already has infrastructure things like `docker-compose`, `dhcp` etc). What do you think?

**Comment by romcheg** [comment]
@wid thanks for the great chart!

I agree with @vi4m that this should perhaps be in the `contrib` catalogue.

At the same time I kindly ask to not merge it right now. I agreed with @szymi- to do a little of a follow-up changes on the top in order to make it a production-grade chart including automatic testing

**Comment by wid** [comment]
Yes, you can do that.

File: charts/ralph/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+
+# Prerequistes
+
+ - Kubernetes cluster >= v1.19.11
+ - Helm 3 
+Find instructions here: https://helm.sh/docs/intro/install/)
+ - KubeDB Community Edition (optional but easier)
+Find instructions here: 
+https://kubedb.com/docs/v2021.06.23/setup/install/community/
```
Comment (stxm) [inline charts/ralph/README.md:9]: Using kubedb seems like a uncommon choice to me. Suggestion to use mysql and redis as subchart.

**Comment by wid** [comment]
Yes, mysql seems a better choice, specially that latest version do not have
basic needed features in the latest versions.

**Pull request closed without merging**